hub/core/sample.py

```diff
@@ -92,7 +92,7 @@ def compressed_bytes(self, compression: str) -> bytes:
             bytes: Bytes for the compressed sample. Contains all metadata required to decompress within these bytes.
         """
 
-        compression = compression.lower()
+        # compression = compression.lower()
```

Comment:
I was facing issues with this line with None compression somewhere in transforms, commented it out temporarily. Is there any reason why we didn't have a check for None here @mccrearyd ? L97 does consider the case of compression being None

Reply:
this lower isn't really needed

Reply:
i would just delete the code

Reply:
```suggestion
```